tools/c7n_azure/tests_azure/tests_resources/test_storage.py

```diff
@@ -362,6 +362,9 @@ def test_enable_log_settings(self):
         }, validate=True)
 
         resources = p.run()
+
+        self.sleep_in_live_mode(30)
```

Comment:
Can you create another issue and tag it here, so we can remove this code when we figure out what is wrong?

Reply:
Created https://github.com/cloud-custodian/cloud-custodian/issues/5309

I'd avoid adding any tags, just lets keep the issue as a reminder. Will add comments to the issue to highlight usages